src/main/java/duke/core/TaskList.java

```diff
@@ -0,0 +1,17 @@
+package duke.core;
+
+import java.util.ArrayList;
```

Comment:
I like how you put a space between the import statements to indicate that they belong to different groups (Java API vs. other package files)!